.github/workflows/workflow.yml

```diff
@@ -278,13 +279,69 @@ jobs:
       - name: Pull SOEF Image
         run: |
           docker pull gcr.io/fetch-ai-images/soef:9e78611 # change this to latest tag
-      - name: Sync AEA loop integration tests
+      - name: Pull fetchd
         run: |
-          tox -e py3.8 -- -m 'sync'  # --aea-loop sync
+          docker pull fetchai/fetchd:0.10.2
+      - name: Pull ganache
+        run: |
+          docker pull trufflesuite/ganache-cli:latest
+      - name: Async integration tests
+        run: tox -e py3.8 -- -m 'integration and not unstable and not ledger' ./tests
```

Comment:
Looks like on release PR, we run all tests  `./tests/*` but also tests in `core_integration_checks` (i.e. `./tests/test_aea_core_packages/*`)

Reply:
this is he point for integration tests
it runs all the integrations

Reply:
Should exclude the `tests/test_aea_core_packages/` folder (it is covered in the below job)

Reply:
done